### ISSUE TITLE
Was it tough to land a job at GitHub?

### ISSUE THREAD

**Issue by ezekg** [mdo/ama#219]
Anything you can tell.
1. How long did the process take?
2. Did you have to apply multiple times for different positions?
3. Did you find it hard waiting for an open position that you thought you would be the best fit in?

**Comment by mdo** [comment]
GitHub reached out to me a long time ago while I was still at Twitter (4-5 year ago). Conversations stalled out and we didn't talk again until quite a bit later, and then I turned them down to finish some stuff up at Twitter. A few months after that I reached out and we made it happen at the end of 2012.

Because of that, I never had to apply for any positions. At the time, I think most of GitHub's hiring was done by reaching out to folks directly instead of having a job board or a place for folks to apply.

I imagine, based on this thread, that you've applied already :). Best advice I can give is to casually reach out to folks when appropriate, be show clear examples of your work, and articulate how awesome you are at what you do :D.

Best of luck!
